app/uk/gov/hmrc/exports/models/declaration/ExportItem.scala

```diff
@@ -63,7 +63,12 @@ object TaricCode {
   implicit val format: OFormat[TaricCode] = Json.format[TaricCode]
 }
 
-case class ItemType(nationalAdditionalCode: Seq[String], statisticalValue: String)
+case class NactCode(nactCode: String)
```

Comment:
I prefer full name `NationalAdditionalCode`.

Reply:
Ok it's name from ticket

Reply:
Was just about to point that out - name matches the UI